src/time_integration/paired_explicit_runge_kutta/methods_PERK3.jl

```diff
@@ -0,0 +1,366 @@
+
+    # Initialize the array of our solution
+    a_unknown = zeros(num_stages - 2)
+
+    # Special case of e = 3
+    if num_stages == 3
+        a_unknown = [0.25]
+    else
+        # Calculate coefficients of the stability polynomial in monomial form
+        consistency_order = 3
+        dtmax = tspan[2] - tspan[1]
+        dteps = 1e-9
+
+        num_eig_vals, eig_vals = filter_eig_vals(eig_vals; verbose)
+
+        monomial_coeffs, _ = bisect_stability_polynomial(consistency_order,
+                                                         num_eig_vals, num_stages,
+                                                         dtmax, dteps,
+                                                         eig_vals; verbose)
+        monomial_coeffs = undo_normalization!(monomial_coeffs, consistency_order,
+                                              num_stages)
+
+        # Solve the nonlinear system of equations from monomial coefficient and
+        # Butcher array abscissae c to find Butcher matrix A
+        # This function is extended in TrixiNLsolveExt.jl
+        a_unknown = solve_a_butcher_coeffs_unknown!(a_unknown, num_stages,
+                                                    monomial_coeffs, cS2, c;
+                                                    verbose)
+    end
+    # Fill A-matrix in P-ERK style
+    a_matrix = zeros(num_stages - 2, 2)
+    a_matrix[:, 1] = c[3:end]
+    a_matrix[:, 1] -= a_unknown
+    a_matrix[:, 2] = a_unknown
+
+    return a_matrix, c
+end
+
+# Compute the Butcher tableau for a paired explicit Runge-Kutta method order 3
+# using provided values of coefficients a in A-matrix of Butcher tableau
+function compute_PairedExplicitRK3_butcher_tableau(num_stages,
+                                                   base_path_a_coeffs::AbstractString;
+                                                   cS2)
+
+    # Initialize array of c
+    c = compute_c_coeffs(num_stages, cS2)
+
+    # - 2 Since First entry of A is always zero (explicit method) and second is given by c_2 (consistency)
+    a_coeffs_max = num_stages - 2
+
+    a_matrix = zeros(a_coeffs_max, 2)
+    a_matrix[:, 1] = c[3:end]
+
+    path_a_coeffs = joinpath(base_path_a_coeffs,
+                             "a_" * string(num_stages) * ".txt")
+
+    @assert isfile(path_a_coeffs) "Couldn't find file"
+    a_coeffs = readdlm(path_a_coeffs, Float64)
+    num_a_coeffs = size(a_coeffs, 1)
+
+    @assert num_a_coeffs == a_coeffs_max
+    # Fill A-matrix in P-ERK style
+    a_matrix[:, 1] -= a_coeffs
+    a_matrix[:, 2] = a_coeffs
+
+    return a_matrix, c
+end
+
+@doc raw"""
+    PairedExplicitRK3(num_stages, base_path_a_coeffs::AbstractString;
+                      cS2 = 1.0)
+    PairedExplicitRK3(num_stages, tspan, semi::AbstractSemidiscretization;
+                      verbose = false, cS2 = 1.0)
+    PairedExplicitRK3(num_stages, tspan, semi::AbstractSemidiscretization;
+                      verbose = false, cS2 = 1.0)
```

Comment:
Am I overlooking something or is this a duplicate?

Reply:
You are right. My bad! I'll correct this.